docs/2_2_common_mechanisms.adoc

```diff
@@ -586,23 +586,26 @@ While the importer is the source of the actual Clock activations, the timing of
 Output Clock::
 [[outputClock, `output Clock`]]
 is a variable of type Clock with <<causality>> == <<output>>.
-The attribute <<interval>> must be <<triggered>> for output Clocks.
+The attribute <<clockType>> must be <<triggered>> for output Clocks.
 The importer calls <<fmi3GetClock>> to inquire the clock activation state.
 
 .Overview of Clock types.
 [[table-overview-clocks]]
-[cols="9,9,16,9,40,40",options="header"]
+[cols="9,9,16,9,40,40"]
 |====
-2+|Clock type
-|<<causality>>
-|<<interval>>
-|Related API calls and arguments
-|Example
+2.2+h|Clock properties
```

Comment:
I think we should discuss in the next meeting if "clockType" is the best alternative to "interval". I guess that people would expect e.g. "periodic" to be a "clock type" because that is the _major_ feature of a clock. On the other hand I think that people are expecting a numerical value when the term "interval" is used.

So I propose "**intervalType**" as alternative to "interval" as it was used in the standard to subsume _constant_, _fixed_, ...